.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,17 @@
+name: Build and Test
+on: [push]
```

Comment:
nit: recommend adding default read-only perms at the top-level of the workflow

```
permissions: read-all
```

Reply:
Done